Automattic/wp-calypso
# Data stores: plans-ui store

Related to https://github.com/Automattic/martech/issues/1491

## Proposed Changes

Adds the data-store `plans-ui` for managing shared state across the Plans UI components. It currently contains a single state property that is needed for the redesign going into https://github.com/Automattic/wp-calypso/pull/73230

## Testing Instructions



* code review
* unit tests
* can see usage in https://github.com/Automattic/wp-calypso/pull/73230

## Pre-merge Checklist
